RecoTracker/FinalTrackSelectors/plugins/MultiTrackSelector.cc

```diff
@@ -561,10 +575,13 @@ void MultiTrackSelector::processMVA(edm::Event& evt, const edm::EventSetup& es,
     return;
   }
 
+  if(!useMVA_[selIndex] && !writeIt)return;
```

Comment:
Hi Slava,
Yes, I can run run2 MC with the code. The issue causing the crash is in AnalyticalTrackSelector: useMVA_ is not initialized properly there. Our tests didn't include the HLT step. I'll commit a fix shortly.